<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>892</width>
    <height>792</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sistema Faculdade</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Informações Faculdade</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>30</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>CNPJ</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="college_name">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="college_cnpj">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>50</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Nome</string>
    </property>
   </widget>
   <widget class="QPushButton" name="college_register_btn">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>50</y>
      <width>83</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Registrar</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="college_address">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>50</y>
      <width>291</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>30</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Endereço</string>
    </property>
   </widget>
   <widget class="QPushButton" name="course_register_btn">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>130</y>
      <width>83</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Registrar</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>110</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Nome</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="course_name">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>130</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Cadastro Curso</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="course_description">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>130</y>
      <width>151</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="course_code">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Código</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>110</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Descrição</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="course_workload">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>130</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>110</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Carga horária</string>
    </property>
   </widget>
   <widget class="QComboBox" name="course_modality_comboBox">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>130</y>
      <width>111</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>110</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Modalidade</string>
    </property>
   </widget>
   <widget class="QPushButton" name="subject_register_btn">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>210</y>
      <width>83</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Registrar</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>260</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Ementa</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="subject_program">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>280</y>
      <width>161</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Cadastro Disciplina</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="subject_name">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>210</y>
      <width>171</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="subject_code">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>111</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Código</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>190</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Nome</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="subject_description">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>210</y>
      <width>231</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_18">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>180</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Descrição</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="subject_bibliography">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>280</y>
      <width>191</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_19">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>260</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Bibliografia</string>
    </property>
   </widget>
   <widget class="QComboBox" name="subject_course_comboBox">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>210</y>
      <width>111</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_20">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>190</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Curso</string>
    </property>
   </widget>
   <widget class="QPushButton" name="class_register_btn">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>370</y>
      <width>83</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Registrar</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="class_maxStudent">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>370</y>
      <width>121</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="class_code">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>370</y>
      <width>111</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_24">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>350</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Código</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_25">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>350</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Disciplina</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_26">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Cadastro Turma</string>
    </property>
   </widget>
   <widget class="QComboBox" name="class_subject_comboBox">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>370</y>
      <width>111</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_27">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>350</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Número de alunos</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_28">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>350</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Turno</string>
    </property>
   </widget>
   <widget class="QComboBox" name="class_shift_comboBox">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>370</y>
      <width>111</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="subject_semester">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>280</y>
      <width>161</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_21">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>260</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Semestre</string>
    </property>
   </widget>
   <widget class="QPushButton" name="show_data_btn">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>730</y>
      <width>191</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Exibir Dados</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_29">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>500</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Contrato</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_30">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>440</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Email</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_31">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>500</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Curso</string>
    </property>
   </widget>
   <widget class="QComboBox" name="teacher_course_combobox">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>520</y>
      <width>111</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="teacher_register_btn">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>460</y>
      <width>83</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Registrar</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="teacher_email">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>460</y>
      <width>121</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="teacher_name">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>460</y>
      <width>111</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_32">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>420</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Cadastro Professor</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_33">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>440</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Nome</string>
    </property>
   </widget>
   <widget class="QComboBox" name="teacher_contract_combobox">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>520</y>
      <width>111</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_34">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>440</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Endereço</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="teacher_address">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>460</y>
      <width>121</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_35">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>440</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>CPF</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="teacher_cpf">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>460</y>
      <width>121</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="teacher_rg">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>460</y>
      <width>121</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_36">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>440</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>RG</string>
    </property>
   </widget>
   <widget class="QComboBox" name="teacher_subject_combobox">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>520</y>
      <width>111</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_37">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>500</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Disciplina</string>
    </property>
   </widget>
   <widget class="QComboBox" name="teacher_class_combobox">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>520</y>
      <width>111</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_38">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>500</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Turma</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_39">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>590</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Email</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="student_email">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>610</y>
      <width>121</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_40">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>590</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>RG</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="student_cpf">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>610</y>
      <width>121</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="student_address">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>610</y>
      <width>121</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="student_class_combobox">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>670</y>
      <width>111</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_42">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>590</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>CPF</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_43">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>650</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Turma</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_44">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>650</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Curso</string>
    </property>
   </widget>
   <widget class="QComboBox" name="student_subject_combobox">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>670</y>
      <width>111</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_45">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>650</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Disciplina</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="student_rg">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>610</y>
      <width>121</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_46">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>590</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Nome</string>
    </property>
   </widget>
   <widget class="QComboBox" name="student_course_combobox">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>670</y>
      <width>111</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="student_register_btn">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>610</y>
      <width>83</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Registrar</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_47">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>570</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Cadastro Aluno</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_48">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>590</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Endereço</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="student_name">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>610</y>
      <width>111</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
